specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -90,10 +90,6 @@ satisfied by any Jakarta EE profile-based product that includes those two
 technologies, thus offering application developers a more complete
 programming model shared across all relevant Jakarta EE profiles.
 
-A separate specification, the Jakarta EE Web
-Profile Specification, defines the Jakarta EE Web Profile, the first
-profile of the Jakarta EE platform.
-
```

Comment:
Why remove this?  If anything I would think it would make sense to include a statement about the Core Profile Specification.

Reply:
I chose to remove it because by doing so we don't need to update it if any other profiles are added. I judge the set of profiles is easily discoverable outside of this text.